rpm/SPECS/openresty-openssl.spec

```diff
@@ -67,18 +67,21 @@ rm -rf %{buildroot}
 %files
 %defattr(-,root,root,-)
 
-%attr(0755,root,root) %{openssl_prefix}/lib/*.so*
-%attr(0755,root,root) %{openssl_prefix}/lib/*/*.so*
-
+%attr(0755,root,root) %{openresty_prefix}/bin/openssl
```

Comment:
Okay, I see you want to include the `openssl` command-line utility. That's fair. But I think we should install it into `%{openssl_prefix}/bin/` instead.